### PR TITLE
Stranglethorn Improvements

### PR DESCRIPTION
**Pull request by geo-tp**
STV
===

Gyll
----
7m_horde_WoWScrnShot_042004_224800

Corporal Kaleb
----

Basilisk
----

Since Stranglethorn is an early zone, basilisk should be present at begining of display_id set.
Here's what we find
basil

All those are way too big, the only one at begining of the set and with a normal scale is 601 (used by another basilisk here). we choose 601 for  Cold eye & trashtrail basilisk.

Gorlash & Mokrash
----
Big Ugly Giant

Naga
----

All nagas should use 4036, we apply it on Naga explorer, Zanzil naga, Lord Sakrasis

Raptor
----

We miss display_id of Jungle stlaker and Thetis, we have no infos about it. On area, there is Young Jungle stalker with a display_id
This display_id 615 is way too big scaled for Young Jungle Stalker.

Probably, Jungle Stlaker and Thethis will use same colors as Young Jungle Stalker. We cant be sure about colors, there are too many raptors in range. But that's maybe why blizzard will update it later to add more color variations.

We can see 3 models that's seems to fit the most, all others are too small or too big.

Capture d’écran_2022-08-13_04-14-50
Capture d’écran_2022-08-13_04-11-19

We apply display_id in taking account of level range. The biggest for named

Bloodsail Warlock
-----
Here's the push on those mobs
Capture d’écran_2022-08-13_02-28-24

Only caster models is unused. In vanilla, blizzard will update only this one. So it's pretty obvious that Bloodsail Warlock should use this display_id.

Troll
----

In 0.5.3, there are actually 3 types of troll : Frostmane (white), Skullsplitter (blue), Bloodscalp (red)

There is a lot of display_id and scale but not so many models. 

Bloodscalp
Capture d’écran_2022-08-13_02-36-12

Model 1:  Beastmaster, Axe thrower (we probably see him on screenshot with tauren below)
Model 2 : Named
Model 3 : Berserker
Model 4 : All others
Model 5 : Witch doctor

We use appropriate scale for each level range.

Skullsplitter
Capture d’écran_2022-08-13_02-41-32

Same here, except there is no model with mask (cant find it in the whole set), we will use model 2 instead for Skullsplitter WItch Doctor, like Zanzil Wich Doctor who use it.

We use appropriate scale for each level range.

Here's some screenshots
mars2002-09
Capture d’écran_2022-08-12_19-18-55
gencon24
novembre2002-19

Grom gol
-----
Here, we can see all mobs with placeholder orc display_id. This model is not part of 0.5.3
album_1_051
E3-72

Probably it was overwrited by this one, this model is known to be a placeholder.

See here (at right, orc arm)
PinkTroll

Probably, NPC & gromgol guards should use 1191 (orc placeholder) & the 2 trolls should use 1478 (unutextured troll)

Here's an old screenshot that shows 2 trolls using placeholder model, and probably get updated with 1478 later.
king_day03_1245

Some mobs have bad spawn in Gromgol. We correct some we can saw on screenshot

Murlocs
-----
We can probably see here stv murlocs push
Capture d’écran_2022-08-13_03-46-39
Capture d’écran_2022-08-13_03-47-00

Basically, all those mobs should use violet murlocs except oracle who should use default blue one. Vanilla do like that but with another color variation for oracle (not present yet)

One of those mobs already use 652. We apply right model with best scale possible for each mobs.

Kurzen
-----

(NO MODIFICATIONS DONE, JUST SOME INFOS)

Here's probably kurzen mobs placeholder, 637 is used by named
Capture d’écran_2022-08-13_03-09-44

### CODE REVIEW

**Comment by geo-tp** [comment]
Fixed Kuzen mobs.

image

637 : Malthus, Kurzen, Elite, Subchief
638 : Commando, Jungle fighter, Shadow hunter, Wrangler
658 : Witch Doctor, Headshrinker, Medicine Man
